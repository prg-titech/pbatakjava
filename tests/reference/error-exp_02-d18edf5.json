{
    "basename": "error-exp_02-d18edf5",
    "cmd": "{infile}",
    "infile": "tests/exp_02/build.sh",
    "infile_hash": "85609461259209558f27232255a25ee5e4cabc8a255d26faba843152",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "error-exp_02-d18edf5.stderr",
    "stderr_hash": "79ce921daf6ccf721e9a11571b333cb160cbebd317bd5f124cb97445",
    "returncode": 4
}