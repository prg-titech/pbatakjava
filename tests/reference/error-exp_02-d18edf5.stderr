tests/exp_02/Test.java:1: error: no solution found in this component
